pkg/chunkenc/gzip.go

```diff
@@ -313,6 +313,13 @@ func (c *MemChunk) SpaceFor(*logproto.Entry) bool {
 	return len(c.blocks) < blocksPerChunk
 }
 
+// Utilization implements Chunk.
+func (c *MemChunk) Utilization() float64 {
+	// Here we're just reporting the block utilization which is not super exciting.  It would be more interesting to see
+	// for each block what %age of c.blocksize was used, but that would require uncompressing the data in every block
+	return float64(len(c.blocks)) / float64(blocksPerChunk)
```

Comment:
you could do it as an average maybe? keep a sum of all the uncompressed bytes and then divide by the (# of blocks * block size)?

Reply:
I think I'm just generally concerned about adding the additional state of "uncompressed size in bytes" to each block.  If you think that knowing the difference between max block size and utilized block size is valuable I will add it.

Reply:
Changing the definition of utilization to be the total number of uncompressed bytes in the chunk divided by the total number of possible uncompressed bytes in the chunk.